### DISCUSSION TITLE
How do I trace the Creation date of a user account?.

### DISCUSSION THREAD

**Discussion by hung-doan** [OrchardCMS/OrchardCore#9951] [Q&A]
Current I got some problem about the troubleshooting/tracking that relates to the user registration & management as I don't know  when the account was created or create by whom to trace the log. 
So my question is: 
- How do I know the creation date of an User Account ?. 
- How do I know the creator of an User Account ?.

Currently the table `[Documentation]` and the content have no information for the auditing ?.

**Comment by hung-doan** [comment]
Any one help in this topic ?

**Reply by jptissot** [reply]
Did you try the AuditTrail module ?

**Reply by hung-doan** [reply]
thanks @jptissot for the reply, It's good to know there is a module support this feature. I'm using the RC2 version, but will checkout the latest version to try that feature.

**Reply by hung-doan** [reply]
btw cheer for the official release  👍 👍
